sktime/forecasting/compose/_multiplexer.py

```diff
@@ -81,6 +78,12 @@ class MultiplexForecaster(
     'arima'
     """
 
+    _tags = {
+        "univariate-only": True,
+        "requires-fh-in-fit": False,
+        "handles-missing-data": False,
+    }
+
```

Comment:
this is not correct - if one of the components requires the `fh` in `fit`, then so does the `MultiplexForecaster` - or, more precisely, if the selected forecaster does.

Question for you, also in general in terms of design for @mloning,@aiwalter, @koralturkk : we seem to have three options here:
* always set the tags to the "worst" - this is the "no-error" setting at class level
* set at construction, based on components but not on value of the multiplexer - this would look at the tags of the components, and set each tag to the "bad" version if it appears in at least one of the components
* set whenever parameters are set using `set_params` - then, set the tags to the tag of the component it points to

I slightly prefer option no.2 (set at construction, look only at tags of components)

Reply:
also, @koralturkk, why did it have the `_OptionalForecastingHorizonMixin` to start with? That looked incorrect, for the same reason as above.

Reply:
this should be changed to `"requires-fh-in-fit" : True`, that's safe until the tags are sorted out.

Reply:
Thank you @fkiraly . I would set it to true

Reply:
I have set it to True, but it fails the test in the true state